lms/djangoapps/ccx/migrations/0004_seed_forum_roles_in_ccx_courses.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from ccx_keys.locator import CCXLocator
+from courseware.courses import get_course_by_id
+from django.db import migrations
+from django.http import Http404
+
+from django_comment_common.models import FORUM_ROLE_ADMINISTRATOR, assign_role
+from django_comment_common.utils import seed_permissions_roles
+
+def seed_forum_roles_for_existing_ccx(apps, schema_editor):
+    """
+    Seed forum roles and make CCX coach forum admin for respective CCX(s).
+
+    Arguments:
+        apps (Applications): Apps in edX platform.
+        schema_editor (SchemaEditor): For editing database schema i.e create, delete field (column)
+
+    """
+    CustomCourseForEdX = apps.get_model("ccx", "CustomCourseForEdX")
+    Role = apps.get_model("django_comment_common", "Role")
+    list_ccx = CustomCourseForEdX.objects.all()
```

Comment:
@smarnach Thanks, I was not aware of that. I added `using(db_alias)`.

The `seed_permissions_roles` code that I was using also queries the database in several places. I decided to inline the code rather than keep using `seed_permissions_roles`, because then I would have to pass `db_alias` around as a parameter.